File: 옛날에 푼거 정리/bak1978.c
//소수찾기
#include <stdio.h>

int main(void)
{
    int i, j, arr[1000] = {0, };
    int n, num, cnt = 0;
    
    arr[0] = 1, arr[1] = 1;
    
    for(j = 2; j < 1000; j++)
        for(i = j * 2; i < 1000; i++)
            if(i % j == 0) arr[i] = 1;
    
    scanf("%d", &n);
    
    for(i = 0; i < n; i++)
    {
        scanf("%d", &num);
        if(arr[num] == 0) cnt++;
    }
    printf("%d", cnt);
    return 0;
}

File: python/bak-step9/bak1978.py
#1978
#소수 찾기

import math

def Prime(k):
     if k == 1:
          return False
     else:
          for i in range(2, int(math.sqrt(k))+1):
               if k%i==0:
                    return False
     return True

num = int(input())
cnt = 0
p = list(map(int, input().split()))

for i in range(0, num):
     if Prime(p[i]):
          cnt+=1
          
print(cnt)
